core/Pos/Crypto/HD.hs

```diff
@@ -124,7 +128,7 @@ unpackHDAddressAttr (HDPassphrase passphrase) (HDAddressPayload payload) = do
 encryptChaChaPoly
     :: ByteString -- nonce (12 random bytes)
     -> ByteString -- symmetric key (must be 32 bytes)
-    -> ByteString -- optional associated data (won't be encrypted)
+    -> ByteString -- optional associated data (for more information see @addrAttrHeader@)
```

Comment:
Nooo, please insert info directly here and get rid of `addrAttrHeader` completely